digit-ui-esbuild/products/dashboard/src/styles/dashboard.css

```
@@ -200,6 +200,82 @@
+  overflow-y: auto;
+  padding-right: 1.75rem;
+}
+
+.dashboard-widget-header-subtitle .dashboard-drag-handle-subtitle{
+  margin: 0px;
+  line-height: 1.25;
+  overflow: visible;
+  text-overflow: unset;
+  white-space: normal;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)